administrator/language/de-DE/com_modules.ini

```diff
@@ -79,6 +79,9 @@ COM_MODULES_FIELD_HEADER_CLASS_DESC="Die CSS-Klasse für den Modul-Header bzw. d
 COM_MODULES_FIELD_MODULE_STYLE_DESC="Den Template-Stil für diese Modulposition überschreiben."
 COM_MODULES_FILTER_SEARCH_DESC="Nach Position filtern."
 COM_MODULES_GENERAL_FIELDSET_DESC="Die Einstellungen um Module zu bearbeiten."
+COM_MODULES_GLOBAL="Zuordnen des Moduls zu Menüeinträgen"
+COM_MODULES_GLOBAL_ASSIGN="Zuordnen zu Menüeinträgen"
+COM_MODULES_GLOBAL_TREE_EXPAND="Expand the Menu Subtrees"
```

Comment:
@chmst do you have a preferred translation for this line? Please make a suggestion.

Reply:
Untermenüs aufklappen
oder 
Baumstruktur der Untermenüs erweitern

Reply:
Baumstruktur haben wir bisher nicht in Joomla